### DISCUSSION TITLE
What HTTP Health Check Methods Do People Use Without NGINX Plus?

### DISCUSSION THREAD

**Discussion by emretorx** [nginx/nginx#366] [General]
Hello everyone,

I am using the open source version of NGINX and wanted to ask if there are any commonly used methods to perform HTTP health checks without NGINX Plus. I know that NGINX Plus offers HTTP health check functionality, but since I don't use NGINX Plus, I was wondering what alternative solutions others in the community use.

Do you use any third-party modules, external tools, or custom scripts for health checking? If so, which ones work best in your experience? Any suggestions would be greatly appreciated!

Thanks in advance!

**Comment by HanadaLee** [comment]
https://github.com/yaoweibin/nginx_upstream_check_module

**Comment by dvershinin** [comment]
https://nginx-extras.getpagespeed.com/nginx-mod/#active-health-checks
